server/plugin.go

```diff
@@ -367,6 +367,43 @@ func (p *Plugin) GetToDo(ctx context.Context, username string, githubClient *git
 	return text, nil
 }
 
+func (p *Plugin) HasNews(info *GitHubUserInfo) bool {
```

Comment:
How `HasUnreads` to be a little more consistent/clear?

Reply:
Sounds good. I will change that.